src/runtime/js/ts.ui/forms/forms-gui@tradeshift.com/spirits/ts.ui.FieldSetSpirit.js

```diff
@@ -38,6 +38,14 @@ ts.ui.FieldSetSpirit = (function() {
 			}
 		},
 
+		onasync: function() {
+			this.super.onasync();
+			var form = this.dom.parent(ts.ui.FormSpirit);
+			if (form && form.css.contains(ts.ui.CLASS_PREVIEW)) {
+				this.element.setAttribute('disabled', 'disabled');
```

Comment:
didn't we want this to be `readonly` ?
the text should still be selectable somehow